cypress/tests/widgets/number.spec.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable cypress/no-unnecessary-waiting */
+describe('Node/-RED Dashboard 2.0 - Number Input Widget', () => {
+    beforeEach(() => {
+        cy.deployFixture('dashboard-number-input')
+        cy.visit('/dashboard/page1')
+    })
+
+    const numberInputOne = '#nrdb-ui-widget-696f064e3552fe7d input[type="text"]'
+
+    // Test case: Renders the Number Input widget correctly
+    it('renders the Number Input widget correctly', () => {
+        cy.get(numberInputOne).should('exist')
+    })
+
+    // Test case: Updates the value correctly
+    it('updates the value correctly and outputs the correct payload', () => {
+        cy.get(numberInputOne).should('exist')
+        cy.get(numberInputOne).clear()
+        cy.get(numberInputOne).type('4')
+        cy.get(numberInputOne).should('have.value', '4')
+        // Wait for the output to be updated as the default delay is 300ms
+        cy.wait(300)
```

Comment:
`wait` is dangerous and can cause inconsistent errors, why no just configure the input to have `0` delay and be able to check this immediately?

_If_ we persist with the wait, wait for longer than the 300ms to give yourself a buffer

Reply:
I've set the widget's default delay to 300ms as per here https://github.com/FlowFuse/node-red-dashboard/pull/1173#issuecomment-2285826655 therefore I've added the above `cy.wait(300)` waiting time before capture the updated value.
But it's better update the value on blue "focus out" and up/down arrow selection

Reply:
Vuetify's Number Input component doesn't support any click events at the moment. Therefore I need to find another solution to tackle this up/down button click by using keyUp event without getting conflict with the existing @keyup:enter
 
https://vuetifyjs.com/en/api/v-number-input/#slots 
 
https://github.com/vuetifyjs/vuetify/blob/master/packages/vuetify/src/labs/VNumberInput/VNumberInput.tsx